typescript/syntax/Statements.sdf3

```diff
@@ -2,19 +2,28 @@ module Statements
 
 imports
 
+  Common
   Declarations
   Expressions
+  Functions
+ 
+context-free priorities
+
+  ReturnStatement.value > ReturnStatement.empty
```

Comment:
How does JavaScript parsers treat that case? Is that disambiguated through layout?